src/__tests__/timerUtils.js

```diff
@@ -0,0 +1,27 @@
+import { setTimeout } from '../helpers/getTimerFuncs';
+
+const TimerMode = {
+  Default: 'default',
```

Comment:
Let's remove default. There are only 2 modes and default will change between v26 and v27.

Reply:
Done